.travis.yml

```diff
@@ -3,25 +3,47 @@ jobs:
   include:
     - name: "Python 3.6.0 on Xenial Linux"
       python: 3.6
+      install: 
+      - pip3 install --upgrade pip  # all three OSes agree about 'pip3'
+      - pip3 install black
```

Comment:
`black` is in the [dev dependencies](https://github.com/jbesomi/texthero/blob/master/setup.cfg), why do we need it there?